requirements.txt

```diff
@@ -1,30 +1,30 @@
-aniso8601==7.0.0
 asgiref==3.2.3
 certifi==2019.11.28
 chardet==3.0.4
 cloudinary==1.20.0
 coverage==5.0.3
 coveralls==1.11.1
 dj-database-url==0.5.0
-Django==3.0.7
+Django==3.0.3
```

Comment:
we bumped this up to 3.0.7 earlier, please change the version back to that, and also update your django version in your mentorpaired dedicated virtualenv.